api/all/src/main/java/io/opentelemetry/api/common/AttributesBuilder.java

```diff
@@ -46,7 +47,7 @@ public interface AttributesBuilder {
    *
    * @return this Builder
    */
-  default AttributesBuilder put(String key, String value) {
+  default AttributesBuilder put(String key, @Nullable String value) {
```

Comment:
Let's add the nullable annotations to the equivalent APIs on Span and LogRecordBuilder APIs

Reply:
Done.